README.adoc

```diff
@@ -241,17 +252,19 @@ In the `start` directory, run the following command to package the `system` micr
 mvn -pl system package
 ```
 
-The `mvn package` command compiles, verifies, and builds the project. The resulting compiled code is packaged into a `war` web archive that can be 
-found under the `system/target` directory. The archive contains the application that is needed to run the microservice on an Open Liberty server, and it is 
-now ready to be injected into a Docker container for deployment.
+The `mvn package` command compiles, verifies, and builds the project.
+The resulting compiled code is packaged into a `war` web archive that can be found under the `system/target` directory.
+The archive contains the application that is needed to run the microservice on an Open Liberty server,
+and it is now ready to be injected into a Docker container for deployment.
 
 == Containerizing the system microservice
 
 === Reusing the Docker daemon
 
-To simplify the local deployment process, the built-in Minishift Docker daemon can be reused. Reusing the Minishift Docker daemon allows your system to 
-connect to it, and use the internal Docker registry. This avoids having to build a Docker registry on your machine. To reuse the Docker daemon, run the 
-following command to point your command-line session to use the Minishift's daemon:
+To simplify the local deployment process, the built-in Minishift Docker daemon can be reused.
```

Comment:
suggested edit lines 264-66

To simplify the local deployment process, you can reuse the built-in Minishift Docker daemon.
Reusing the Minishift Docker daemon allows you to use the internal Docker registry, so you don't have to build a Docker registry on your machine.